components/card/card.js

```diff
@@ -56,7 +56,7 @@ class Card extends FocusMixin(RtlMixin(LitElement)) {
 			 */
 			target: { type: String, reflect: true },
 			/**
-			 * Accessible text for the card (will be announced when AT user focuses)
+			 * ACCESSIBILITY: REQUIRED: Accessible text for the card for screen reader users
```

Comment:
```suggestion
			 * ACCESSIBILITY: REQUIRED: Accessible text describing the card to screen reader users
```
Same thing as above though, not sure if describing implies too much text